create.go

```diff
@@ -52,7 +57,124 @@ var createCommand = cli.Command{
 		},
 	},
 	Action: func(context *cli.Context) error {
-		// TODO
-		return nil
+		return create(context.String("container-id"),
+			context.String("bundle"),
+			context.String("console"),
+			context.String("pid-file"))
 	},
 }
+
+func create(containerID, bundlePath, console, pidFilePath string) error {
+	// Checks the MUST and MUST NOT from OCI runtime specification
+	if err := checkCreateParamsIntegrity(containerID, bundlePath); err != nil {
+		return err
+	}
+
+	podConfig, err := getPodConfig(bundlePath, containerID, console)
+	if err != nil {
+		return err
+	}
+
+	pod, err := vc.CreatePod(podConfig)
+	if err != nil {
+		return err
+	}
+
+	shimInfo, err := getShimInfo(pod)
+	if err != nil {
+		return err
+	}
+
+	// Start the shim to retrieve its PID.
+	pid, err := startShim(shimInfo)
+	if err != nil {
+		return err
+	}
+
+	if err := storeShimPid(pod, pid); err != nil {
+		return err
+	}
+
```

Comment:
Is there any case where we're going to start a shim without storing its PID ? it seems to me this is an atomic operation and you could have `startShim(pod)` do `getShimInfo()`, then start the shim process and finally `storeShimPid()`. `startShim(pod)` would return the shim PID.